oneview_redfish_toolkit/api/metadata.py

```diff
@@ -21,7 +21,7 @@
 import xml.etree.ElementTree as ET
```

Comment:
Copyright of this file should be updated.

Reply:
Done.